src/schemas/StageStepMetric.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
+properties:
+  stage_id:
```

Comment:
duplicate? we already have the info in the parent element (Stage.yaml)

Reply:
done